pkg/controllers/management/usercontrollers/controller.go

```diff
@@ -293,6 +294,12 @@ func (c *ClusterLifecycleCleanup) createCleanupJob(userContext *config.UserConte
 		Labels:       map[string]string{"cattle.io/creator": "norman"},
 	}
 
+	cleanupImage := settings.AgentImage.Get()
+	systemDefaultRegistry := settings.SystemDefaultRegistry.Get()
+	if systemDefaultRegistry != "" {
+		cleanupImage = fmt.Sprintf("%s/%s", systemDefaultRegistry, cleanupImage)
+	}
```

Comment:
Wondering if we can just use existing `resolve.Resolve` here (`resolve.ResolveWithCluster` may be an option as well but as far as I remember we don't support cluster-level private registries for imported clusters)?
https://github.com/rancher/rancher/blob/dd23578d3e3499f54d80c4ed88cc52c44ee69539/pkg/image/resolve.go#L42-L47

Reply:
Yeah that would work too, I'll update the PR. Also I agree, I do not believe we support cluster level registries for imported clusters.